tests/zfs-tests/tests/perf/perf.shlib

```diff
@@ -122,33 +122,33 @@ function do_fio_run_impl
 	# disable client cache for reads.
 	if [[ $NFS -eq 1 ]]; then
 		export DIRECT=1
-		do_setup_nfs $script
+		do_setup_nfs "$script"
 	else
 		export DIRECT=0
 	fi
 
 	# This will be part of the output filename.
-	typeset suffix=$(get_suffix $threads $sync $iosize)
+	typeset suffix=$(get_suffix "$threads" "$sync" "$iosize")
 
 	# Start the data collection
-	do_collect_scripts $suffix
+	do_collect_scripts "$suffix"
 
 	# Define output file
 	typeset logbase="$(get_perf_output_dir)/$(basename \
-	    $SUDO_COMMAND)"
+	    "$SUDO_COMMAND")"
 	typeset outfile="$logbase.fio.$suffix"
 
 	# Start the load
 	if [[ $NFS -eq 1 ]]; then
-		log_must ssh -t $NFS_USER@$NFS_CLIENT "
+		log_must ssh -t "$NFS_USER"@"$NFS_CLIENT" "
```

Comment:
Here and below, do we need to close quotes around `@`?

Reply:
No, what happened here is shellcheck was being conservative and trying to avoid changing any behavior. But for the purposes of this logging, "$NFS_USER"@"$NFS_CLIENT" and "$NFS_USER@$NFS_CLIENT" should be identical.

Reply:
It is not (just) a logging, but passing an argument to `ssh`.  But still it is expected to be one argument, as I understand.